ObjectWeb.Asm/Tree/AbstractInsnNode.cs

```diff
@@ -206,7 +207,7 @@ internal static LabelNode[] Clone(List<LabelNode> labels, IDictionary<LabelNode,
             var clones = new LabelNode[labels.Count];
             for (int i = 0, n = clones.Length; i < n; ++i)
             {
-                clones[i] = clonedLabels.GetValueOrNull(labels[i]);
```

Comment:
Couldn't you make these method take a `IReadOnlyDictionary<LabelNode, LabelNode>` and then use the`GetValueOrDefault` extension?

Reply:
no